cove_360/lib/threesixtygiving.py

```diff
@@ -389,7 +389,7 @@ def process(self, grant, path_prefix):
             count_failure = False
             for num, organization in enumerate(grant['recipientOrganization']):
                 has_postal_code = organization.get('postalCode')
-                has_location_data = organization.get('location') and organization.get('location').get('geoCode') and organization.get('location').get('geoCodeType')
+                has_location_data = organization.get('location') and any(location.get('geoCode') and location.get('geoCodeType') for location in organization.get('location'))
                 complete_recipient_org_data = has_postal_code and has_location_data
```

Comment:
I think that making this `has_postal_code or has_location_data` instead would also address https://github.com/OpenDataServices/cove/issues/801

`self.message` in line 405 may need a minor tweak too:
https://github.com/OpenDataServices/cove/blob/3ba413119b4e43de2fdcff51628b5f59e0d2a942/cove_360/lib/threesixtygiving.py#L405